src/main/scala/is/hail/expr/ir/MatrixIR.scala

```diff
@@ -610,12 +610,14 @@ case class MatrixAggregateRowsByKey(child: MatrixIR, expr: IR) extends MatrixIR
 
     val nCols = prev.nCols
 
+    val (minColType, minColValues, rewriteIR) = PruneDeadFields.minimizeColValues(prev, expr)
```

Comment:
I think `pruneColValues` is a clearer and more consistent name (minimizing is overloaded).